specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -0,0 +1,13585 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v2.0",
+    "title": "Speech Services API v2.0",
+    "description": "Speech Services API v2.0.",
+    "contact": {
+      "name": "Cognitive Services - Speech Services",
+      "email": "crservice@microsoft.com"
+    }
+  },
+  "host": "westus.cris.ai",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/api/speechtotext/v2.0/accuracytests": {
+      "get": {
+        "tags": [
+          "Custom Speech accuracy tests:"
+        ],
+        "summary": "Gets the list of accuracy tests for the authenticated subscription.",
+        "operationId": "AccuracyTests_List",
```

Comment:
Please update to AccuracyTest_List to make it inconsistent with the rest of the methods, like AccuracyTest_Create, since it's causing SDK generation issues in Java for example https://dev.azure.com/azure-sdk/public/_build/results?buildId=17447&view=logs&jobId=ef1a6f21-3eb6-5439-fa7a-dcd9cf589dd1&taskId=7d38bc47-3f36-578f-2b95-a19eb7179ec4&lineStart=155&lineEnd=156&colStart=1&colEnd=1

Reply:
Btw, I will not expose this to customers just yet, or at least until we fix the error. I just want to complete this so we get on a roll, get to know your process and start making progress.

Reply:
@PanosPeriorellis The fact of merging it to master means it's exposed to customers. Why do you want to merge this PR to master in the current state?
If you're intending to auto-generate SDKs we need to make sure that what we merge, meets the quality bar for SDKs. Java SDK generation is currently failing due to the Model name inconsistencies and the way return codes are specified for negative responses are no good for SDK auto generation. I understand the APIs are in GA, but the changes requested affect SDK generation.